docs/UsersGuide/source/BuildingRunningTesting/ContainerQuickstart.rst

```diff
@@ -271,9 +271,9 @@ The ``wflow_<platform>`` modulefile will then output instructions to activate th
 .. code-block:: console
 
    Please do the following to activate conda:
-       > conda activate workflow_tools
+       > conda activate regional_workflow
 
-then the user should run ``conda activate workflow_tools``. This will activate the ``workflow_tools`` conda environment. The command(s) will vary from system to system, but the user should see ``(workflow_tools)`` in front of the Terminal prompt at this point.
+then the user should run ``conda activate regional_workflow``. This will activate the ``regional_workflow`` conda environment. The command(s) will vary from system to system, but the user should see ``(regional_workflow)`` in front of the Terminal prompt at this point.
```

Comment:
@christinaholtNOAA This was updated to `workflow_tools` ahead of the release, so you can go ahead and change it back to that.

Reply:
Sure thing!